jobs/ml-ops/terraform/providers.tf

```diff
@@ -0,0 +1,16 @@
+provider "scaleway" {
+  region     = var.region
+  access_key = var.access_key
+  secret_key = var.secret_key
+  project_id = var.project_id
```

Comment:
nitpitck: FYI, doing this includes those variables in the Terraform state file. It's a bit safer to configure the provider via env vars or the scw config file.

```suggestion
```

Reply:
These variable are not only used to configure the provider but also by resources (private registries, secrets for jobs/containers). However, the Terraform state file is git-ignored as a guardrail.